cli/xterminal/terminal.go

```diff
@@ -0,0 +1,40 @@
+//go:build !windows
```

Comment:
Thoughts on putting this in our `pty` package instead? Feels a bit weird having two terminal packages.

Reply:
Sounds good

Reply:
This is 👍🏻, so feel free to ignore suggestion in: https://github.com/coder/coder/pull/12990#discussion_r1569215285